blockchain/src/main/java/com/tangem/blockchain/blockchains/hedera/HederaBuiltTransaction.kt

```diff
@@ -1,8 +1,8 @@
 package com.tangem.blockchain.blockchains.hedera
 
-import com.hedera.hashgraph.sdk.TransferTransaction
+import com.hedera.hashgraph.sdk.Transaction
 
-data class HederaBuiltTransaction(
-    val transferTransaction: TransferTransaction,
+internal data class HederaBuiltTransaction<T : Transaction<T>>(
+    val transaction: Transaction<T>,
```

Comment:
Разве не должно быть так, ведь `T` это и так `Transaction`?
```suggestion
    val transaction: T,
```